pkg/sarif.go

```diff
@@ -403,10 +403,9 @@ func (r *ScorecardResult) AsSARIF(showDetails bool, logLevel zapcore.Level,
 
 	// nolint
 	for i, check := range r.Checks {
-
-		doc, exists := checkDocs.Checks[check.Name]
-		if !exists {
-			panic(fmt.Sprintf("invalid doc: %s not present", check.Name))
+		doc, e := checkDocs.GetCheck(check.Name)
+		if e != nil {
+			panic(fmt.Sprintf("GetCheck: %v: %s", e, check.Name))
```

Comment:
Return error instead of panic?

Reply:
This should never happen and should be caught in the `validate` part that we run during the PR validation. I suppose I can still exit gracefuly.

Reply:
I would still recommend that we use error instead of panic. This helps new contributors good practice and raising panic causes the application to terminate.